code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -31,7 +31,7 @@
 /datum/reagent/drug/nicotine
 	name = "Nicotine"
 	id = "nicotine"
-	description = "Slightly reduces stun times. If overdosed it will deal toxin and oxygen damage."
+	description = "Slightly increases stamina regenration If overdosed it will deal toxin and oxygen damage."
```

Comment:
Slightly increases stamina regeneration. If overdosed, it will deal toxin and oxygen damage.